packages/account/src/Configs/get-mt5-status-badge-config.tsx

```diff
@@ -10,72 +10,54 @@ const getMT5StatusBadgeConfig = (mt5_account_status: TMT5AccountStatus) => {
+                icon_size: '19',
             };
         case MT5_ACCOUNT_STATUS.FAILED:
             return {
                 text: (
                     <Localize
-                        i18n_default_text='<0>Verification failed.</0> <1>Why?</1>'
-                        components={[
-                            <Text key={0} weight='bold' size='xxxs' color='var(--status-danger)' />,
-                            <Text
-                                key={1}
-                                className='link-verification-failed'
-                                onClick={() => {
-                                    //TODO: default jurisdiction;
-                                }}
-                            />,
-                        ]}
+                        i18n_default_text='Failed'
+                        className='link-verification-failed'
+                        onClick={() => {
+                            //TODO: default jurisdiction;
+                        }}
                     />
                 ),
-                icon: 'IcRedWarning',
+                icon: 'IcMt5Failed',
+                icon_size: 19,
             };
         case MT5_ACCOUNT_STATUS.NEEDS_VERIFICATION: {
             return {
-                text: (
-                    <Localize
-                        i18n_default_text='<0>Needs verification.</0><1>Verify now</1>'
-                        components={[
-                            <Text key={0} weight='bold' size='xxxs' color='var(--status-info)' />,
-                            <Text
-                                key={1}
-                                className='link-need-verification'
-                                onClick={() => {
-                                    //TODO: default jurisdiction;
-                                }}
-                            />,
-                        ]}
-                    />
-                ),
-                icon: 'IcAlertInfo',
+                text: <Localize i18n_default_text='Needs Verification' />,
+                icon: 'IcMt5Verification',
+                icon_size: '19',
             };
         }
         case MT5_ACCOUNT_STATUS.MIGRATED_WITH_POSITION:
             return {
                 text: <Localize i18n_default_text='<0>No new positions</0>' components={[BadgeTextComponent]} />,
                 icon: 'IcAlertWarning',
+                icon_size: '19',
             };
         case MT5_ACCOUNT_STATUS.MIGRATED_WITHOUT_POSITION:
             return {
                 text: <Localize i18n_default_text='<0>Account closed</0>' components={[BadgeTextComponent]} />,
                 icon: 'IcAlertWarning',
+                icon_size: '19',
             };
         case MT5_ACCOUNT_STATUS.UNDER_MAINTENANCE:
             return {
                 text: <Localize i18n_default_text='<0>Server maintenance</0>' components={[BadgeTextComponent]} />,
                 icon: 'IcAlertWarning',
+                icon_size: '19',
```

Comment:
![Screenshot 2024-09-09 at 10 00 37 AM](https://github.com/user-attachments/assets/2a739cc9-7499-4298-8455-31459dedb0e3)
can you double check the icon sizes? as per design it should be 12 and 18

Reply:
![Screenshot 2024-09-09 at 4 32 06 PM](https://github.com/user-attachments/assets/59460425-c9bb-4e4b-a761-7ea5a2c1cf85)

yes the icon is of that size

Reply:
i have made it 18